src/components/GenericErrorView/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
This is pretty subjective, but I feel like this component might be named `GenericErrorPage` and moved to `src/pages/` instead of `src/components/`. The components directory should generally be used for reusable components that can be used in pages, whereas this is a one-off error page that won't be reused.

Reply:
Let me know if that makes sense

Reply:
That does make sense. I thought if pages don't have routes we don't put them into pages. I'll update.